scope_test.go

```diff
@@ -71,6 +71,25 @@ func TestScopedOperations(t *testing.T) {
 		assert.Error(t, c.Invoke(useB))
 	})
 
+	t.Run("private provides doesn't depend on invoke order", func(t *testing.T) {
+		c := digtest.New(t)
+
+		c.RequireProvide(func() string { return "container" })
+
+		child := c.Scope("child")
+		child.RequireProvide(func() string { return "child" })
+
+		verifyContainerStr := func(s string) {
+			assert.Equal(t, "container", s)
+		}
+		verifyChildStr := func(s string) {
+			assert.Equal(t, "child", s)
+		}
+
+		c.RequireInvoke(verifyContainerStr)
+		child.RequireInvoke(verifyChildStr)
```

Comment:
```suggestion
		child.RequireInvoke(verifyChildStr)
		c.RequireInvoke(verifyContainerStr)
```

Reply:
I don't think this will test the fix, calling the invoke from the child scope first won't cause problem. Am I missing something?